pkg/chaosdaemon/httpchaos_server.go

```diff
@@ -108,19 +116,19 @@ func (s *DaemonServer) applyHttpChaos(ctx context.Context, logger logr.Logger, i
 
 	req, err := http.NewRequest(http.MethodPut, "/", bytes.NewReader(config))
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrap(err, "fail to create http request")
```

Comment:
Please refer this docs: https://github.com/YangKeao/rfcs/blob/refine-error-handling/text/2021-09-27-refine-error-handling.md#error-wrapping

Reply:
Not using `fail to` to describe this error.

Reply:
Fixed